app/controllers/application_controller.rb

```diff
@@ -50,7 +50,7 @@ def default_sort_options
   end
 
   def sort(collection)
-    ModelSorter.sort(collection, params, default_sort_options)
+    RailsSort.sort(collection, params, default_sort_options)
```

Comment:
I see this has been added, but I don't see any mention of it in dependencies. Was it added previously?